pango/Gir.toml

```diff
@@ -9,6 +9,7 @@ min_cfg_version = "1.40"
 use_gi_docgen = true
 single_version_file = true
 deprecate_by_min_version = true
+trust_return_value_nullability = true
```

Comment:
Also remove all the `nullable` overrides from this file then :)

Reply:
Ok!